src/test/java/io/vertx/test/core/LocalEventBusTest.java

```diff
@@ -1275,5 +1275,29 @@ public void testCloseCallsEndHandlerWithRegistrationContext() throws Exception {
     closeVertx();
     await();
   }
+
+  @Test
+  public void testHandlerRegistrationDoesNotCancelTimer0() throws Exception {
+    HandlerRegistration<String> handlerRegistration = new HandlerRegistration<String>(vertx, null, null, null, false,
```

Comment:
HandlerRegistration is an internal class - it would be better to trigger the functionality using the public API - i.e. create a message consumer and unregister it.